variables.tf

```diff
@@ -85,11 +90,15 @@ variable "skip_final_snapshot" {
 
 variable "subnet_ids" {
   type        = list(string)
-  default     = []
   description = "List of subnet IDs to deploy Redshift in"
 }
 
+variable "vpc_id" {
+  type        = string
+  description = "ID of the VPC to deploy Redshift in"
+}
```

Comment:
What value does this add when you can also distill this from the given subnets? Seems just redundant data right, or does it solve a problem?

Reply:
It solves this problem on a fresh deploy
```Error: Invalid index

  on .terraform/modules/datalayer.redshift/main.tf line 7, in data "aws_subnet" "selected":
   7:   id = var.subnet_ids[0]
    |----------------
    | var.subnet_ids is empty list of string

The given key does not identify an element in this collection value.
```